conformance/zip.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+mkdir -p artifacts
+touch artifacts/junit_01.xml
+
+input="./bazel-out/k8-fastbuild/testlogs/conformance/ct/test.xml"
```

Comment:
Add a comment describing what we're expecting to be at the top of the test.xml file.  Note that XML can be substantially reformatted without a change to its meaning, so relying on newlines being in a particular place is an extra assumption - leave plenty of hints for future debuggers.